warehouse/ingest-core/src/main/java/datawave/ingest/mapreduce/handler/edge/evaluation/EdgePreconditionCacheHelper.java

```diff
@@ -16,12 +16,12 @@ public class EdgePreconditionCacheHelper {
     
     private JexlEngine engine;
     
-    public EdgePreconditionCacheHelper() {
-        createEngine();
+    public EdgePreconditionCacheHelper(EdgePreconditionArithmetic arithmetic) {
+        createEngine(arithmetic);
     }
     
-    private void createEngine() {
-        this.setEngine(new JexlEngine(null, new MultiMapArithmetic(false), null, null));
```

Comment:
Can we delete the MultiMapArithmetic class?

Reply:
Can we focus on the urgent bugfix nature of this ticket?